README.md

```diff
@@ -66,34 +66,34 @@ By the end of the course, students will be able to:
+| 101        | Instructor         | Joel Ostblom         | joel.ostblom[-at-]ubc.ca            | Fri 4-5pm       | Zoom                 |
+| 003        | TA                 | Andal Abro Khan      | n/a                                 |                 |                      |
+| 003        | TA                 | Jossie Jiang         | n/a                                 | Thu 11-12pm     | Zoom                 |
+| 003        | TA                 | Ali Mehrabian        | n/a                                 | Thu 1-2pm       | Zoom                 |
+| 003        | TA                 | Eros Rojas           | n/a                                 |                 |                      |
+| 004        | TA                 | Lily Xie             | n/a                                 |                 |                      |
+| 004        | TA                 | Jeyshinee Pyneeandee | n/a                                 | Fri 5-6pm       | Zoom                 |
+| 004        | TA                 | Linda Han            | n/a                                 | Fri 1-2pm       | ESB 1041             |
+| 004        | TA                 | Wasay Hayat          | n/a                                 |                 |                      |
+| R sections | TA                 | Mahsa Zarei          | n/a                                 | Mon 7-8pm       | Zoom                 |
+| 008        | TA                 | Angelique Clara      | n/a                                 |                 |                      |
+| 008        | TA                 | Richard Yang         | n/a                                 |                 |                      |
+| 008        | TA                 | Nelson Li            | n/a                                 |                 |                      |
+| 008        | TA                 | Kevin Wang           | n/a                                 | Thu 1-2pm       | ESB 1041             |
+| 009        | TA                 | Moira Renata         | n/a                                 | Wed 6-7pm       | Zoom                 |
+| 009        | TA                 | Samuel Leung         | n/a                                 | Tue 10-11am     | ESB 1041             |
+| 009        | TA                 | Kaylee (Min-Er) Li   | n/a                                 |                 |                      |
+| 009        | TA                 | Eric Li              | n/a                                 | Fri 10-11am     | Zoom                 |
+| 101        | TA                 | Atabak Eghbal        | n/a                                 | Mon 5-6pm       | Zoom                 |
+| 101        | TA                 | Feifei Yang          | n/a                                 | Thu 3-4pm       | Zoom                 |
+| 101        | TA                 | Indu Kant Deo        | n/a                                 |                 |                      |
```

Comment:
these changes were just the result of my linter fixing the whitespace in the chart.